python/plugins/processing/algs/qgis/CheckValidity.py

```diff
@@ -158,7 +158,7 @@ def doCheck(self, method, parameters, context, feedback):
 
                     reason = "\n".join(reasons)
                     if len(reason) > 255:
-                        reason = reason[:252] + '...'
+                        reason = reason[:252] + '…'
```

Comment:
I gave it a try https://github.com/qgis/QGIS/pull/5012/files
But there is so many in the code. I wonder if it makes sense to produce an error if someone writes ... in a comment.